src/controllers/expenses.controllers.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const expensesService = require('../services/expenses.services');
+
+const getExpenses = (req, res) => {
```

Comment:
You use it as `expenseConrollers.[your_method]` so we already know that it related to expense

```suggestion
const get = (req, res) => {
```